### PR TITLE
Translate Dutch

### PR DESCRIPTION
**Pull request by tvlooy**

### CODE REVIEW

File: releases/8.0/common.php [hunk 1]
```diff
@@ -64,6 +64,7 @@ function language_chooser(string $currentLang): void {
         'fr'    => 'French',
         'ru'    => 'Russian',
         'zh'    => 'Chinese Simplified',
+        'nl'    => 'Nederlands',
```
Comment (tvlooy) [inline releases/8.0/common.php:67]: if #368 is accepted. Otherwise it's "Dutch" here

**Review by heiglandreas — COMMENTED** [review]
Why are we giving russian and chinese in the respective language but the others in english?

**Comment by cmb69** [comment]
It seems to me we're using the native languages (e.g. "Nederlands" would be "Dutch" in English), but only Russian and Simplified Chinese use different scripts than Latin.

**Comment by heiglandreas** [comment]
Forget my comment. I have no idea what I have seen or what I mixed it up with. 🤦

**Comment by carusogabriel** [comment]
@WyriHaximus Wanna review this one? :D

**Comment by WyriHaximus** [comment]
@carusogabriel sure, hold me `::`!

File: releases/8.0/nl.php [hunk 2]
```diff
@@ -0,0 +1,504 @@
...
+    <div class="php8-compare__content">
+      <p>In plaats van een controle op null uit te voeren kan je nu een ketting van oproepen vormen met de null-veilige operator.
+        Wanneer één expressie in de ketting faalt, zal de rest van de ketting niet uitgevoerd worden en is het resultaat van
+        de hele ketting null.</p>
+    </div>
+  </div>
+
+  <div class="php8-compare">
+    <h2 class="php8-h2" id="saner-string-to-number-comparisons">
+      Verstandige tekst met nummer vergelijkingen
```
Comment (WyriHaximus) [inline releases/8.0/nl.php:301]: `Verstandigere` maybe?

**Comment by cmb69** [comment]
But please note that a Dutch translation currently wouldn't work, see https://news-web.php.net/php.webmaster/28963.

File: releases/8.0/nl.php [hunk 3]
```diff
@@ -0,0 +1,504 @@
...
+        </li>
+        <li>
+          Mixed type <a href="https://wiki.php.net/rfc/mixed_type_v2">RFC</a>
+        </li>
+        <li>
+          Static return type <a href="https://wiki.php.net/rfc/static_return_type">RFC</a>
+        </li>
+        <li>
+          Types voor interne functies
+          <a href="https://externals.io/message/106522">Email thread</a>
```
Comment (WyriHaximus) [inline releases/8.0/nl.php:411]: `Email draadje` maybe?

**Review by WyriHaximus — APPROVED** [review]
Aside from two small notes LGTM :+1: 

P.S. Still weird to read this in Dutch

File: releases/8.0/nl.php (hunk 3 above)
Comment (tvlooy) [reply releases/8.0/nl.php]: hard to translate, I take the suggestion!

File: releases/8.0/nl.php (hunk 2 above)
Comment (tvlooy) [reply releases/8.0/nl.php]: better!

**Comment by tvlooy** [comment]
> P.S. Still weird to read this in Dutch

It is :D Weird, but not obscure. It was fun to translate

**Comment by tvlooy** [comment]
> But please note that a Dutch translation currently wouldn't work, see https://news-web.php.net/php.webmaster/28963.

oh ... I was not aware of that. No drama, I just jumped on it in all my PHP 8 excitement :)

**Comment by WyriHaximus** [comment]
> > P.S. Still weird to read this in Dutch
> 
> It is :D Weird, but not obscure. It was fun to translate

It was also fun to read, think "hah this could be better" only to find out it was already good :+1: !

**Comment by cmb69** [comment]
Thanks for the translation!

**Pull request closed without merging**